book/src/external-snapshotter.md

```diff
@@ -0,0 +1,26 @@
+# CSI `external-snapshotter`
+
```

Comment:
Should we indicate at the top of each sidecar's page the alpha/beta/ga status?

Reply:
Good idea. Done